### PR TITLE
support option to embed schema

### PR DESCRIPTION
**Pull request by mengqiy**
Support an option to embed schema.

### CODE REVIEW

File: embed.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+func embedDefinition(def *Definition, refs Definitions) {
+	if def == nil {
+		return
+	}
+
+	if len(def.Ref) > 0 {
+		refName := strings.TrimPrefix(def.Ref, defPrefix)
+		if strings.Contains(refName, "RollingUpdate") {
+			fmt.Println("hahahahaha")
```
Comment (redborian) [inline embed.go]: It is good to have `TODO`s here :)

**Comment by redborian** [comment]
I think once we fully support `embed` option, we should add it to the `Readme` file

**Review by redborian — APPROVED** [review]

File: embed.go (hunk 1 above)
Comment (mengqiy) [reply embed.go]: I was for debugging.
I have dropped it now.
Thanks for catching it.

**Comment by mengqiy** [comment]
Pushed minor changes to address the comment.
Merging.

**Pull request merged**